### ISSUE TITLE
It says 'restricted content, coins required for access.' even though it's 0 YEN chapter.

### ISSUE THREAD

**Issue by SparklerMan** [catsital/pyccoma#1]
As put in the instructions, I installed the package and then proceeded to to execute the following. (in the python shell)
> from pyccoma import Scraper
> pyc = Scraper()
> pyc.login(my email, password)
> pyc.fetch('https://piccoma.com/web/viewer/75078/1979424')

And then got this:
Restricted content, coins required for access.

**Comment by catsital** [comment]
May I know what version is installed on your system? I think this is caused by the same parsing error encountered on <0.2.2. Please try to update to >0.3.0 for the fix and let me know if the issue still occurs. Thank you!

**Comment by SparklerMan** [comment]
It is on version <0.3.0>, and I've actually found out the issue, as you know when not in japan (piccoma) doesn't work, and currently I am not in japan, and so because of that it seemed that it couldn't download it from my local internet connection, however it had thought it was due to the fact that I didn't have coins. Now that I used a VPN to direct to a japan IP, it works. Thank you for your time.
